src/test/java/pages/BasePage.java

```diff
@@ -0,0 +1,48 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class BasePage {
+    public WebDriver driver;
+    public WebDriverWait wait;
+
+    public BasePage(WebDriver driver) {
+        this.driver = driver;
+        wait = new WebDriverWait(driver, 10);
+    }
+
+    public void waitVisibility(By elementBy) {
+        wait.until(ExpectedConditions.visibilityOfAllElementsLocatedBy(elementBy));
```

Comment:
visibilityOfAllElementsLocatedBy is the plural, but it's only possible for one element, this could be confusing.